behind-the-veil-siteroot/imports/ui/components/message/Conversation.jsx

```diff
@@ -0,0 +1,81 @@
+
+export const Conversation = ({ user }) => {
+
+    const [formValue, setFormValue] = useState('');
+    const [messages, setMessages] = useState([]);
+    const conversationRef = useRef(null);
+
+    const sendMessage = (e) => {
+        e.preventDefault();
+        if (formValue.trim() === '') return;
+        setMessages(prevMessages => [...prevMessages, { text: formValue, sender: 'me' }]);
+        setFormValue('');
+        let heightToScroll = conversationRef?.current.scrollHeight + 50
+        setTimeout(() => {
+            conversationRef?.current.scrollTo({ left: 0, top: heightToScroll, behaviour: "smooth" })
+        }, 5)
+
+    };
+
+    return (
+        <div className="flex flex-col fixed top-0 bottom-0 left-0 right-0">
+            <div className='ml-4 w-11/12 message-receiver-name-text border-b-2 pt-3 pb-1 mb-8 pl-6 border-main-blue'>{user.name}
+            </div>
+            <div className="flex-1 overflow-y-auto p-4" ref={conversationRef}>
+                <div>
+                    {user.messages.map((message, index) => (
+                        <div key={index}>
+                            <div className={`${message.sender === 'me' ? 'flex  justify-end' : 'flex'}`}>
+                                <div> {/* wrapping the profilephoto inside this div helps to keep the size consistent when smaller screen */}
+                                {message.sender !== 'me' && (
+                                    <ProfilePhoto className={`${message.sender === 'me' ? 'order-last flex' : ''} min-win-[10%]`}></ProfilePhoto>
+                                )}
+                                </div>
+                                <Card className={`my-2 rounded-3xl max-w-[80%] border-transparent ${message.sender === 'me' ? ' bg-main-blue' : 'bg-light-grey'} `}>
+                                    {message.text}
+                                </Card>
+                            </div>
+                        </div>
+
+                    ))}
+
+                    {messages.map((message, index) => (
+                        <div key={index} className="flex justify-end">
+                            <Card className={`my-2 rounded-3xl max-w-[80%] border-transparent ${message.sender === 'me' ? ' bg-main-blue' : 'bg-light-grey'} `}>
+                                {message.text}
+                            </Card>
+                        </div>
+                    ))}
+                </div>
+            </div>
+
+            <form className="w-full px-8 py-6 " onSubmit={sendMessage}>
+                <div className=' relative w-full flex'>
+                    <input
+                        value={formValue}
+                        onChange={(e) => setFormValue(e.target.value)}
+                        placeholder="Type a message..."
+                        className=" w-full px-4 py-3 rounded-3xl border border-gray-300 focus:outline-none focus:border-blue-500"
```

Comment:
could we add larger padding for just the right side of this input field? this is what happens with the current padding for long messages: 
![image](https://github.com/Monash-FIT3170/Behind-The-Veil/assets/95511987/63d4cb97-3fbd-4a59-a604-e2a941ef4309)

Reply:
added more padding